lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -148,15 +148,18 @@ def get_config(self, flags=None):
             self._device_configs[cmd] = cfg
             return cfg
 
-    def run_commands(self, commands, check_rc=True):
+    def run_commands(self, commands, check_rc=True, return_error=False):
```

Comment:
Per our discussion, I'd like to see `check_rc` used instead of adding `return_error`